recipes/pasta/0.2/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+out_dir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+pasta_tools_dev_dir=$PREFIX/share
+share_dir=$PREFIX/share
+
+mkdir -p $out_dir
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/pasta
+
+cp -R ./* $out_dir
+
+unamestr=`uname`
+if [ $unamestr == 'Linux' ];
+then
+    wget https://github.com/smirarab/sate-tools-linux/archive/master.tar.gz
```

Comment:
This is not reproducible as it is the master branch.
Can we not have a separate package for this one?